DANIEL_PIGA.md

```diff
@@ -0,0 +1,5 @@
+Hello I am Daniel Piga.
```

Comment:
You probably noticed that one line break is not enough to create a new paragraph in Markdown - it still renders everything in this file as a single paragraph. 

It would make it easier for a reader and provide more structure to the document if you split the content into [multiple paragraphs](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#paragraphs).